wsi/drm/drmdisplay.cpp

```diff
@@ -575,9 +575,20 @@ void DrmDisplay::UpdateDisplayConfig() {
   }
   flags_ |= DRM_MODE_ATOMIC_ALLOW_MODESET;
   SetDisplayAttribute(modes_[config_]);
+  vsync_period_ = modes_[config_].vrefresh;
   SPIN_UNLOCK(display_lock_);
 }
 
+bool DrmDisplay::GetDisplayVsyncPeriod(uint32_t* outVsyncPeriod) {
+  SPIN_LOCK(display_lock_);
+  if (modes_.empty()) {
+    SPIN_UNLOCK(display_lock_);
+    return false;
```

Comment:
if modes is empty, you can return the default one.
it is how we handle headless case https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/drm/drmdisplay.cpp#L273